examples/compare_license_texts.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+
+# Parses Two License Text Strings passed into them 
+
+if __name__ == '__main__':
+    from normalize_license_text.normalize_class import NormalizeText
+    from compare_template_text.compare_normalized_files import CompareNormalizedFiles
+    from compare_template_text.normalize_template_text import NormalizeTemplate
```

Comment:
Remove unnecessary imports from all the files.

Reply:
@rtgdk did you setup the project using pip install -e .

Reply:
yeah, here's the pip list output
`pip list
Package                     Version Location                                 
--------------------------- ------- -----------------------------------------
pip                         20.0.2  
setuptools                  46.1.3  
spdx-python-licensematching 1.0     /Users/rlodha/spdx-python-licensematching
wheel                       0.34.2  
`

Reply:
@rtgdk The problem thats happening is that the normalize_class file is not able to read the file path of "equivalentwords.txt". To import the file path I have written
`equivalent_words_file = PACKAGE_PATH + "/resources/equivalentwords.txt"
`
in line 7 of normalize_class.py file where PACKAGE_PATH is the variable for the home directory path which gets initialized in configuration after setting up.
Can you show the output of the PACKAGE_PATH, it should show in your case "/Users/rlodha/spdx-python-licensematching"

Reply:
You can even run the file configuration/config.py to get value of PACKAGE_PATH.